src/SFA.DAS.Assessor.Functions/Domain/Print/Services/NotificationService.cs

```diff
@@ -5,55 +5,44 @@
 using SFA.DAS.Assessor.Functions.ExternalApis.Assessor.Constants;
 using SFA.DAS.Assessor.Functions.ExternalApis.Assessor.Types;
 using SFA.DAS.Assessor.Functions.Infrastructure;
-using SFA.DAS.Notifications.Api.Client;
-using SFA.DAS.Notifications.Api.Types;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace SFA.DAS.Assessor.Functions.Domain.Print.Services
 {
     public class NotificationService : INotificationService
-    {
-        private readonly INotificationsApi _notificationsApi;
+    {        
         private readonly ILogger<NotificationService> _logger;
         private readonly SftpSettings _sftpSettings;
         private readonly IAssessorServiceApiClient _assessorServiceApi;
 
-        public NotificationService(INotificationsApi notificationsApi,
+        public NotificationService(
             ILogger<NotificationService> logger,
             IOptions<SftpSettings> options,
             IAssessorServiceApiClient assessorServiceApi)
-        {
-            _notificationsApi = notificationsApi;
+        {            
             _logger = logger;
             _sftpSettings = options?.Value;
             _assessorServiceApi = assessorServiceApi;
         }
 
+        
         public async Task Send(int batchNumber, List<CertificateResponse> certificateResponses, string certificatesFileName)
         {
-            var emailTemplate = await _assessorServiceApi.GetEmailTemplate(EMailTemplateNames.PrintAssessorCoverLetters);
+            var emailTemplateSummary = await _assessorServiceApi.GetEmailTemplate(EMailTemplateNames.PrintAssessorCoverLetters);
+
+            var recipients = emailTemplateSummary.Recipients.Split(';').Select(x => x.Trim());
```

Comment:
I though that the point of using the send email request in the functions application was to avoid having to split the recipients in the function app - that could be encapsulated in the request?